react/components/OrderInfo/Payment/PaymentMethod.js

```diff
@@ -14,22 +15,24 @@ const paymentGroupSwitch = (payment) => {
 }
 
 const PaymentMethod = ({ payment }) => (
-  <div className="flex flex-column justify-around mr7-ns">
-    <p className="t-heading-6">{paymentGroupSwitch(payment.group)}</p>
-    <PaymentFlagPicker paymentSystem={payment.paymentSystem}>
-      {FlagComponent =>
-        FlagComponent && (
-          <div className="h2">
-            <FlagComponent />
-          </div>
-        )
-      }
-    </PaymentFlagPicker>
-    <p>Final {payment.lastDigits}</p>
-    <div>
-      <FormattedPrice value={payment.value} currency="BRL" />{` (${payment.installments}x)`}
-    </div>
-  </div>
+  <CurrencyContext.Consumer>
```

Comment:
Try to follow this pattern:

https://github.com/vtex/admin-subscription-app/blob/1afc4e490d1a4d38e6652478697d02440d333a36/react/context/index.js#L6